torch/onnx/_internal/exporter/_torchlib/ops/hop.py

```diff
@@ -92,3 +92,53 @@ def higher_order_cond(
             (), else_node.outputs, nodes=[else_node], name=false_func.name
         ),
     )
+
+
+@onnx_impl(torch.ops.higher_order.scan, no_compile=True)
+def higher_order_scan(
+    body_func: ir.Function,
+    scan_inits: Sequence[ir.Value],
+    scan_inputs: Sequence[ir.Value],
+    additional_inputs: Optional[Sequence[ir.Value]] = None,
+    reverse: bool = False,
+) -> Sequence[ir.Value]:
+    node_inputs = [
+        *[
+            # inputs are clone to make sure they have different names
+            # from additional inputs which could be the same:
+            # one input can be a scanned input and an additional input
+            ir.node("Identity", [i], num_outputs=1).outputs[0]
```

Comment:
```suggestion
            ir.Node("Identity", [i], num_outputs=1).outputs[0]
```

Maybe this for consistency.